docs: remove duplicate row in table about expression and filte

karolinelien opened the pull request:
remove duplicate row in table about "Reference information: Expression and filter examples per value type"

ceciliapersson commented:
@jason-p-pickering Should I merge this PR?

jason-p-pickering commented:
Not yet.

On Thu, Sep 29, 2016, 8:50 PM Cecilia Persson <notifications@github.com>
wrote:

> @jason-p-pickering <https://github.com/jason-p-pickering> Should I merge
> this PR?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dhis2/dhis2-docs/pull/247#issuecomment-250557257>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AC1J_egso_wqjje9qg_1Ab5zwq_yrP3Lks5qvAhxgaJpZM4KKO-Y>
> .
>

ceciliapersson commented:
ok

karolinelien committed:
remove duplicate row in table about expression and filte

remove duplicate row in table about "Reference information: Expression and filter examples per value type"
